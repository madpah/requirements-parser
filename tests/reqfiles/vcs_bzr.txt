-e bzr+http://bzr.myproject.org/MyProject/trunk#egg=MyProject
-e bzr+sftp://user@myproject.org/MyProject/trunk#egg=MyProject
-e bzr+ssh://user@myproject.org/MyProject/trunk#egg=MyProject
-e bzr+ftp://user@myproject.org/MyProject/trunk#egg=MyProject
-e bzr+https://bzr.myproject.org/MyProject/trunk@2019#egg=MyProject
-e bzr+http://bzr.myproject.org/MyProject/trunk@v1.0#egg=MyProject

bzr+http://bzr.myproject.org/MyProject/trunk#egg=MyProject
bzr+sftp://user@myproject.org/MyProject/trunk#egg=MyProject
bzr+ssh://user@myproject.org/MyProject/trunk#egg=MyProject
bzr+ftp://user@myproject.org/MyProject/trunk#egg=MyProject
bzr+https://bzr.myproject.org/MyProject/trunk@2019#egg=MyProject
bzr+http://bzr.myproject.org/MyProject/trunk@v1.0#egg=MyProject
