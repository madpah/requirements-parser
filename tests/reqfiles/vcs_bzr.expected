[
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+http://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+http://bzr.myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+sftp://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+sftp://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null 
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+ssh://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+ssh://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+ftp://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+ftp://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+https://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+https://bzr.myproject.org/MyProject/trunk@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "bzr+http://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "-e bzr+http://bzr.myproject.org/MyProject/trunk@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+http://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+http://bzr.myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+sftp://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+sftp://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+ssh://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+ssh://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+ftp://myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+ftp://user@myproject.org/MyProject/trunk#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+https://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+https://bzr.myproject.org/MyProject/trunk@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "bzr+http://bzr.myproject.org/MyProject/trunk", 
  "extras": [], 
  "vcs": "bzr", 
  "path": null, 
  "line": "bzr+http://bzr.myproject.org/MyProject/trunk@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }
]
